### PR TITLE
Use of image picker for pick from gallery instead of file_picker.

### LINKED ISSUE #701: [Bug] -  Build fail due to use of compatible file_picker version fetched with any with other dependency
- **Issue Description:**
Due to the use of any version in the ```pubspec.yaml``` it fetches the version compatible with other dependencies. Resulting in fetching the previous versions which are compatible with the other dependencies of ```path_provider```,```graphql_flutter``` and ```http``` etc.

- **Issue Severity:** <High or Medium orLow>
High, this causes build fail and hinders other issues to be solved.

- **Environment Details:** <Add details about environment where this issue is found.>
```create_organization.dart``` ,```update_organization.dart``` and ```generated_plugin_registrant.dart```

- **Observed Behaviour:** <Add details about your observation related to this issue.>
Build fail due to the functions used are not found in the previous version of dependency fetched due to any keyword

- **Expected Behaviour:** <Add what would like to see once this is issue is fixed.>
```image_picker``` dependency used is capable of picking images from gallery itself then we can reduce our dependency on the ```file_picker``` plugin just to only fetch a image from gallery. Though if we need to use the ```file_picker``` then we can update the conflicting dependency stated above in a separate issue.

- **Steps to reproduce issue:** <Provide detailed steps to reproduce the issue.>
Rebase your fork and let the dart analyser run or build the project after pub get.

- **Does this issue need immediate attention?** <If yes. Please explain in detail>
Yes, this help in fixing the build fail of the project.

- **Are you willing to work on this issue:** <Yes or No. If you are working on other issue then issue will get assigned to someone else for getting work done faster.>
Yes, I would like to work on the issue.

- **Other information:** <Add extra information about this PR here>
Dependencies  ```path_provider```,```graphql_flutter``` and ```http``` etc. need to be updated in new issue.

### PR DESCRIPTION
**Pull request by CyberWake**
**What kind of change does this PR introduce?**
Fixes the build issue of the project as stated in the issue #701 

**Did you add tests for your changes?**
dependency provides it inbuild.

**If relevant, did you update the documentation?**
Documentation need not be changed.

**Summary**
This PR fixes #701. This also reduces our dependency on ```file_picker``` as of now which is only used to pick image from gallery feature which could be done by ```image_picker``` dependency as well.

**Does this PR introduce a breaking change?**
No, this PR does not bring any breaking changes.

**Other information**
Closes #701

### CODE REVIEW

**Comment by CyberWake** [comment]
@Sagar2366 sir please review this PR.

File: pubspec.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies:
   data_connection_checker: ^0.3.4
   datetime_picker_formfield: ^1.0.0
   email_validator: ^1.0.4
-  file_picker: any
+  #file_picker: any
```
Comment (Sagar2366) [inline pubspec.yaml:28]: remove unused library

Comment (Sagar2366) [reply pubspec.yaml:28]: @CyberWake please remove commented unused library.

**Comment by palisadoes** [comment]
We will soon be enforcing a 300 line limit on Dart files. Please consider ways to reduce the sizes of the files you edit while maintaining functionality in future. PRs reviewed on or after May 15 will be rejected if this requirement is not met.

**Pull request merged**